scenarios/sns_secrets/terraform/ec2.tf

```diff
@@ -0,0 +1,74 @@
+data "aws_ami" "amazon_linux" {
+  most_recent = true
+  owners      = ["amazon"]
+
+  filter {
+    name   = "name"
+    values = ["amzn2-ami-hvm-*-x86_64-gp2"]
+  }
+}
+
+resource "aws_instance" "cg-sns-instance" {
```

Comment:
It looks like the EC2 instance is only used for the cronjob, it might make more sense to use a lambda function for this.

Could also remove the VPC elements too

Reply:
I agree with you, but as soon as I touch terraform it breaks :rofl: - Since the functionality remains the same either way, I'm going to have it use the EC2 and VPC. I verified all the whitelisting works well. I also decreased the EC2 size so it's less resources as well.

Reply:
👍